### ISSUE TITLE
Beacon node panic on testnet

### ISSUE THREAD

**Issue by benjaminion** [prysmaticlabs/prysm#4088]
I had one beacon node and one validator running happily for about 5 days.  However, the beacon node suffered a panic that also caused the validator to fail. They ran on for another day before I noticed, but did not recover (same error keeps occurring).

Set up: I'm running the docker images on a QNAP NAS box exactly as per the initial instructions.

```
REPOSITORY                                TAG                 IMAGE ID            CREATED             SIZE
gcr.io/prysmaticlabs/prysm/beacon-chain   latest              0d644ba659e4        49 years ago        57.2MB
gcr.io/prysmaticlabs/prysm/validator      latest              3b689e4c1230        49 years ago        48.8MB
```

(Seems to be a unix time issue on the created time 😂)

Log file extracts from the time of the failure are here: https://gist.github.com/benjaminion/d4cfb037418e3129c4bf5ce98b75336d

I had a quick skim for duplicates, and didn't see anything identical, but I may have missed things. LMK what else you need.

**Comment by terencechain** [comment]
Thanks for opening the issue Ben!

Looks like your node is requesting parent block and not able to get it through peers. We'll take deeper look at this today

**Comment by prestonvanloon** [comment]
FYI: about unix CREATED time https://github.com/bazelbuild/rules_docker/issues/52#issuecomment-308490078

TL;DR = you can't have reproducible builds and timestamps ¯\_(ツ)_/¯

**Comment by prestonvanloon** [comment]
@benjaminion can you print the full image ID?

```
docker images --no-trunc
```

I'm having a hard time determining what version / commit you were running with just that information.

**Comment by benjaminion** [comment]
Here you go:
```
REPOSITORY                                TAG                 IMAGE ID                                                                  CREATED             SIZE
gcr.io/prysmaticlabs/prysm/beacon-chain   latest              sha256:0d644ba659e4d906d0ec7845acc8a99a0255bbef5a0aa7e3a014a15169b2f04e   49 years ago        57.2MB
gcr.io/prysmaticlabs/prysm/validator      latest              sha256:3b689e4c12303f2f6bdb1e887c4f59656136499896bb4a67750a1619e3f86943   49 years ago        48.8MB
```

This might help as well:
```
# curl http://localhost:7000/eth/v1alpha1/node/version

{"version":"Git commit: 5acc362f7e954e6369d5a709937165728f93eb99. Built at: 2019-11-16 00:26:26+00:00"}
```

**Comment by prestonvanloon** [comment]
Thanks this really helps!

**Comment by prestonvanloon** [comment]
So this is the offending line of code: https://github.com/prysmaticlabs/prysm/blob/5acc362f7e954e6369d5a709937165728f93eb99/beacon-chain/sync/subscriber_beacon_blocks.go#L19

And this panic was fixed in https://github.com/prysmaticlabs/prysm/pull/4042

Closing for now, but please reopen if it persists on a recent commit.
